src/config.cc

```diff
@@ -181,10 +185,10 @@ Config::Config() {
       {"rocksdb.level_compaction_dynamic_level_bytes",
         false, new YesNoField(&RocksDB.level_compaction_dynamic_level_bytes, false)},
   };
-  for (const auto &wrapper : fields) {
-    auto field = wrapper.field;
+  for (auto &wrapper : fields) {
+    auto &field = wrapper.field;
     field->readonly = wrapper.readonly;
-    fields_.insert({wrapper.name, field});
+    fields_.insert({wrapper.name, std::move(field)});
```

Comment:
```suggestion
    fields_.insert({std::move(wrapper.name), std::move(field)});
```
We can also move the name string 🤣